contents/blog/rome-hackathon.md

```diff
@@ -0,0 +1,76 @@
+---
+date: 2022-10-05
+title: "All the cool things we built at our Rome hackathon"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+categories: ["Inside PostHog"]
+---
+
+As an all-remote team, we know how important getting together in person is (we’ve [written about this before](/blog/asynchronous-remote-companies)). Our [product analytics team](/handbook/people/team-structure/product-analytics) (plus Luke, Annika, Cory, and Lottie) recently got back from an offsite in Rome. On top of eating a lot of pizza and gelato, refocusing and aligning work, and 360 feedback sessions, they had a hackathon and built a bunch of new things. 
+
+We are showcasing them here. Some are sneak previews, others are works-in-progress, and they’ll likely change. We wanted to give you an idea of what our team actually does at an offsite hackathon.
```

Comment:
Would be good to add a way people can give feedback here

Reply:
What is the recommended way to do this?

Reply:
Added Slack link, anything else?

Reply:
Added a bit about raising an issue in the repo